constrain/library/vav_turndown_during_reheat.py

```diff
@@ -62,6 +62,8 @@ def verify(self):
             mean_no_reheat_ratio = self.df.loc[
                 ~self.df["reheat_coil_flag"], "V_dot_VAV_ratio"
             ].mean()
-            self.df["result"] = mean_reheat_ratio < mean_no_reheat_ratio
+            self.df["result"] = mean_reheat_ratio < (
+                mean_no_reheat_ratio - self.get_tolerance("ratio", "flow")
+            )
```

Comment:
I don't think that this is necessary, but perhaps if we assume a default tolerance of 0 it won't hurt and could give flexibility to the user to use a tolerance when need be.